src/omnisharpServer.ts

```diff
@@ -57,9 +135,30 @@ export abstract class OmnisharpServer {
 	private _setState(value: ServerState) : void {
 		if (typeof value !== 'undefined' && value !== this._state) {
 			this._state = value;
-			this._fireEvent('stateChanged', this._state);
+			this._fireEvent(Events.StateChanged, this._state);
 		}
 	}
+    
+    private _recordDelay(path: string, elapsedTime: number) {
```

Comment:
Might be obvious to other people, but I am having trouble figuring out what 'path' means.